README.md

```diff
@@ -66,7 +66,7 @@ Cohere Endpoint | Function
 /classify | co.classify()
 
 ## Models
-To view an up-to-date list of available models please consult the models section in the [platform](https://os.cohere.ai). To get started try out `large` for Generate, `medium` for Classify or `small` for Embed.
+To view an up-to-date list of available models please consult the models section in the [platform](https://os.cohere.ai). To get started omit `model` from requests to use the default model sizes.
```

Comment:
"To get started, omit" sounds a bit awkward. Maybe something like this? "When you call Cohere's APIs we decide on a good default model for your use-case behind the scenes. The default model is great to get you started, but in production environments we recommend that you specify the model size yourself via the `model` parameter. Learn more about the available models here..."